web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc

```diff
@@ -8,6 +8,16 @@ author = "Thomas Fischer"
     parent = "workflows"
 +++
 
+## Unsupported Processes
+So far, the parallel FEM scheme with PETSc cannot be used by the following processes:
+
+- HydroMechanics (including that with LIE)
+- ThermoHydroMechanics
```

Comment:
```suggestion
- ThermoMechanics
- ThermoHydroMechanics
- RichardsMechanics
```

Reply:
@endJunction Thanks. ThermoMechanics is "luck" in this issue, it uses the same order of elements for both T and M. 

Took the other suggestions by amending the commit.